k8s/helm chart/argilla/templates/argilla-server-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argilla-server-deployment
+  labels:
+    app: argilla-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: argilla-server
+  template:
+    metadata:
+      labels:
+        app: argilla-server
+    spec:
+    {{- if .Values.enableOptionalResource }}
+      initContainers:
+        - name: wait-for-elasticsearch
+          image: alpine/curl:latest
+          command: [ "sh", "-c",
+            "ELASTICSEARCH_URL={{ .Values.argilladeployment.elasticurl }}; status_code=$(curl -s -o /dev/null -w '%{http_code}' $ELASTICSEARCH_URL);
+          while [ $status_code -ne 200 ]; do sleep 5; status_code=$(curl -s -o /dev/null -w '%{http_code}' $ELASTICSEARCH_URL);
+          echo Sleeping...; done; echo Elasticsearch Connected " ]
+    {{- end }}
+      containers:
+        - name: argilla-server
+          image: argilla/argilla-server:latest
+          env:
+            - name: {{ .Values.argilladeployment.name }}

```

Comment:
This env name is required by Argilla and wouln't change between deployments. Leaving it as `ARGILLA_ELASTICSEARCH` should suffice.

Reply:
https://github.com/bikash119/argilla/blob/70059f21bfd977ee726978ba74f44a4d3d026767/examples/deployments/k8s/argilla-chart/templates/deployment.yaml#L33